closed/openj9_version_info.h.in

```diff
@@ -38,7 +38,7 @@
 #define OPENJ9_TAG                "@OPENJ9_TAG@"
 #define OPENJDK_SHA               "@OPENJDK_SHA@"
 #define OPENJDK_TAG               "@OPENJDK_TAG@"
-#define OPENJDK_VERSION_SHORT     "@VERSION_SHORT@"
+#define OPENJDK_VERSION_STRING    "@VERSION_STRING@"
```

Comment:
Did you intend to remove `J9VERSION_STRING` with the same value? Later? Or should we just use `J9VERSION_STRING` instead of adding `OPENJDK_VERSION_STRING`?

Reply:
I left J9VERSION_STRING in for now, as removing it will break OpenJ9 compiles. We can remove it later after OpenJ9 stops using it. We could just use J9VERSION_STRING, but I think it's badly named and might be confusing.